lib/sentry/config.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Sentry.Config do
```

Comment:
@samphilipd thanks for the feedback.  this should still be the case.  was there a regression that broke runtime configuration changes?